leshan-server-core/src/main/java/org/eclipse/leshan/server/bootstrap/BootstrapConfigStore.java

```diff
@@ -27,7 +27,8 @@
      * 
      * @param endpoint the endpoint of the device.
      * @param deviceIdentity the {@link Identity} the device.
```

Comment:
Should be renamed in the javadoc as well.

Reply:
the parameter renaming was not intended. I revert it (sender => deviceIdentity)